test/test.js

```diff
@@ -956,6 +956,35 @@ describe('sanitizeHtml', function() {
       }), '<span style="color:blue"></span>'
     );
   });
+  it('Should ignore styles when options.parseStyleAttributes is false', function() {
+    assert.equal(
+      sanitizeHtml('<span style=\'color: blue; text-align: justify\'></span>', {
+        allowedTags: false,
+        allowedAttributes: {
+          span: [ 'style' ]
+        },
+        parseStyleAttributes: false
+      }), '<span style="color: blue; text-align: justify"></span>'
+    );
+  });
+  it('Should throw an error if both allowedStyles is set and  && parseStyleAttributes is set to false', function() {
+    try {
+      sanitizeHtml('<span style=\'color: blue; text-align: justify\'></span>', {
+        allowedTags: false,
+        allowedAttributes: {
+          span: ['style']
+        },
+        allowedStyles: {
+          p: {
+            'text-align': [/^justify$/]
+          }
+        },
+        parseStyleAttributes: false
+      });
```

Comment:
Need an "assert(false)" at the end of the "try" block so we confirm we didn't get past it.